bok_choy/page_object.py

```diff
@@ -564,3 +564,28 @@ def wait_for_element_invisibility(self, element_selector, description, timeout=6
 
         """
         self.wait_for(lambda: self.q(css=element_selector).invisible, description=description, timeout=timeout)
+
+    @unguarded
+    def scroll_to_element(self, element_selector, timeout=60):
+        """
+        Scrolls browser such that the element specified appears at the top. Before scrolling, waits for
+        the element to be present.
+
+        Example usage:
+
+        .. code:: python
+
+            self.scroll_to_element('.far-down', 'Scroll to far-down')
+
+        Arguments:
+            element_selector (str): css selector of the element.
+            timeout (float): Maximum number of seconds to wait for the Promise to be satisfied before timing out
+
+        """
+        # Ensure element exists
+        msg = "wait for element '{element}' to be present".format(element=element_selector)
```

Comment:
I think the pattern we have been using for descriptions in the `wait_for` calls is to describe the state that is being waited for.  (One of the examples in this file is `Submit Button is Present`.)  The output on failure then says "Broken Promise: Submit Button is Present".